### ISSUE TITLE
Workload -> Security context: fsgroup missing 

### ISSUE THREAD

**Issue by depeo** [rancher/dashboard#4099]
The ability to add  fsgroup under security context is missing in the new UI.

Can it be added, since the old Cluster Manager is gone now?

**Comment by nwmac** [comment]
See below - Next to 'Run as User ID' add 'Filesystem Group' in the UI.

The property is 'securityContext.fsGroup'

![Screenshot 2021-09-02 at 15.43.14.png](https://zube.io/files/rancher/ea534ad198b54ac0cf15571a2c7333cc-screenshot-2021-09-02-at-15-43-14.png)

**Comment by davidnuzik** [comment]
@nwmac Just want to double-check that this placement in the UI is okay. Your comment seemed to suggest the field should go just next to (such as to the right) the "Run as User ID" field. Is it being it's own separated field like pictured okay?

![image.png](https://zube.io/files/rancher/c58d7a3008fd276bccbb661399d2ad7a-image.png)

**Comment by davidnuzik** [comment]
cc: @gaktive for my last comment. This is probably fine but I wanted to doubly check (UX - is the location of this field okay?)

**Comment by gaktive** [comment]
@davidnuzik it's not a blocker so it's fine for now. We'll see if any UX complaints come in but with the header, it should be clear what goes there and what it relates to. Feel free to file a bug if usage matches possible intent based on direction.

**Comment by davidnuzik** [comment]
**My checks PASSED**
 
**Reproduction Steps:**

Not required.

-----
 
**Validation Environment:**

Rancher version: v2.6-head `ba43472` pulled at 9/21/21 10:43am
Rancher cluster type: single-node docker install
Docker version: 20.10
 
Downstream cluster type: rke2
Downstream K8s version: v1.21.4+rke2r2
 
**Validation steps:**

1. Create a workload, go to the Security Context section
Confirmed this field now exists:

![image.png](https://zube.io/files/rancher/c58d7a3008fd276bccbb661399d2ad7a-image.png)
2. Confirmed I can enter a filesystem group in this field and that the containers runs with this filesystem group. I entered various valid values (integers) from 1 to 10 and this appeared to work without any issues.
Web request/response upon submission of deployment create form appears correct.
